### ISSUE TITLE
hello, when i train with my own datasets, the loss can't decline (Iter 25000). I want to detect cups in an image(1920x1080),but the cups is very small.   Is this the problem here？

### ISSUE THREAD

**Issue by Rguoo** [motokimura/PyTorch_Gaussian_YOLOv3#10]

**Comment by motokimura** [comment]
Hi. I guess YOLOv3 has enough potentials to detect small objects as long as you give enough amount of data. 
How many training samples do you have? Can you upload 2 or 3 of them here?

**Comment by Rguoo** [comment]
Hi, i have 361 images and about 2000 objects. Is the output result correct with a negative number of xy and wh?

**Comment by motokimura** [comment]
> Is the output result correct with a negative number of xy and wh?

It's correct. loss for xy and wh can be negative.

How does the val AP change during the training?
YOLOv3 loss declines very slowly, so you should pay more attention to val APs.

**Comment by Rguoo** [comment]
ok , fine. I will try again use another datasets. Thanks a lot.
